src/main/java/uk/gov/companieshouse/overseasentitiesapi/converter/Transformer.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.overseasentitiesapi.converter;
+
+import org.bson.Document;
+import uk.gov.companieshouse.overseasentitiesapi.model.SchemaVersion;
+
+/**
+ * Implemented by model transformers identified by their version.
+ */
+public interface Transformer {
```

Comment:
I think the name Transformer is a bit too generic, as the methods you have to implement are related to schema versions, so maybe 'SchemaVersionTransformer' or 'DocumentVersionTransformer' would be more descriptive.

Reply:
👍 I'd suggest we go with just `DocumentTransformer`. Schema/DocumentVersionTransformer suggests it could be the version that is being transformed.